app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ApplicationSubmissionsPage.scala.html

```diff
@@ -66,14 +66,20 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-1">@{messages("applicationsubmi
 
     @if(model.currentSubmission.nonEmpty) {
       <dl class="govuk-summary-list govuk-!-margin-bottom-0">
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.requestedby")}</dt>
-        <dd id="requesterEmail" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.requesterEmail}</p></dd>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.requesteddate")}</dt>
-        <dd id="requestedTimestamp" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.timestamp}</p></dd>
-      </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.requestedby")}</dt>
+          <dd id="requesterEmail" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.requesterEmail}</p></dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.requesteddate")}</dt>
+          <dd id="requestedTimestamp" class="govuk-summary-list__value"><p class="govuk-body">@{model.currentSubmission.get.timestamp}</p></dd>
+        </div>
+        @if(model.responsibleIndividualEmail.nonEmpty) {
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@{messages("applicationsubmissions.responsibleindividualemail")}</dt>
+            <dd id="requestedTimestamp" class="govuk-summary-list__value"><p class="govuk-body">@{model.responsibleIndividualEmail}</p></dd>
```

Comment:
Wont this output `Some(emailAddress)` to screen?

Reply:
No, it outputs:

![image](https://user-images.githubusercontent.com/91944208/169323566-8f85ce49-92f6-40a0-afa1-cc6bb57c1e02.png)

Reply:
k

Reply:
Also the id for this email element is requestedTimestamp which it probably shouldn't be

Reply:
Fixed